.gitignore

```diff
@@ -23,3 +23,5 @@
 
 # temporary files from the twisted.trial test runner
 _trial_temp/
+
+.vscode/
```

Comment:
What's this for?

Reply:
Thats to gitignore the settings files generated by VSCode, I can remove it if its an issue

Reply:
That should go into your personal global gitignore: https://stackoverflow.com/questions/7335420/global-git-ignore#7335487

I know we have a few IDE-specific things in this gitignore already, but they should be removed as well.